flask_ldap_login/__init__.py

```diff
@@ -208,9 +208,11 @@ def connect(self):
             if isinstance(opt, str):
                 opt = getattr(ldap, opt)
 
-            if isinstance(value, str):
-                value = getattr(ldap, value)
-
+            try:
+                if isinstance(value, str):
+                    value = getattr(ldap, value)
```

Comment:
@Back2Basics, This will leave `value` undefined in the case of an exception. 

Also, this solution can be more simply written by adding a third argument to `getattr`:

```py
value = getattr(ldap, value, None)
```